src/proposals/mips/mip-b10/MTokens.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
Confirmed these match the recommendations from Warden Finance at https://forum.moonwell.fi/t/mip-b10-onboard-reth-as-collateral-on-base-deployment/672